libraries/WiFi/src/utility/nodriver.h

```diff
@@ -48,6 +48,10 @@ class NoDriver {
         return true;
     }
 
+    PinStatus interruptMode() {
```

Comment:
Could we standardize these, please?  Some are normal fcns, some are `constexpr`, some are `static constexpr` now.  Compiled code should probably not differ much(at all?) but it's weird to have them all over the map.

OTW, LGTM to do this than your #2036

Reply:
I did the new method in every class to match the style of interruptIsPossible method in that class. there would be more to cleanup. for example interruptIsPossible doesn't do anything useful

Reply:
OK, no worries!